cli/azd/cmd/root.go

```diff
@@ -147,23 +147,21 @@ func NewRootCmd(staticHelp bool, middlewareChain []*actions.MiddlewareRegistrati
 		}).
 		UseMiddleware("hooks", middleware.NewHooksMiddleware)
 
+	login := newLoginCmd("")
+	login.Hidden = true
```

Comment:
What is happening here? Is this to support `azd auth login` as well as `az login` ?

Reply:
Correct. This is a version of aliasing that doesn't require any additional code duplication. We're simply reattaching the command both at `root` and at `auth`.